dashboard/index.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require_once("DataProcessor.php");
+
+$containerDataProcessor = new DataProcessor();
+$data = $containerDataProcessor->process();
+
+$tbodyData = '';
+foreach ($data as $dataEntry) {
+    if (count($dataEntry['domains']) < 1) {
+        $tbodyData .= '<tr><td>' . $dataEntry['name'] . '</td><td>' . $dataEntry['domains'][0] . '</td></tr>';

```

Comment:
if count is smaller than 1, then index 0 is not given
+ plus, you dont need to check this, as a foreach already gives you all necessary information about emptiness

Reply:
I am checking the number of domains of one data row, it has here nothing to do with the index of the for loop